services/register.js

```diff
@@ -0,0 +1,47 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+    region: 'us-east-1'
+})
+const utils = require('../utils/utils')
+const bcrypt = require('bcryptjs')
+
+const dynamodb = new AWS.DynamoDB.DocumentClient();
+const userTable = 'jinemister-users';
+
+async function register(userInfo) {
+    const name = userInfo.name;
+    const email = userinfo.email;
+    const unsername = userinfo.username;
```

Comment:
there is a typo here, which might create bugs. Also, wanted to make sure if you can add unit tests as well along with this for register? Thanks!

Reply:
Yeah, thanks for the catch. Resolved the typo.